start_notebook_server.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/bash
```

Comment:
Are you OK with changing this to 
`#!/usr/bin/env bash` ?
bash is at `/bin/bash` on Mac OS, and there's an outside chance that someone other than me may eventually want to use this tool on a Mac, rather than a Linux box.

Reply:
Sure thing, thanks.  I've been trying to train myself into the whole UsrMerge thing and use `/usr/bin/...` rather than `/bin/...` for scripting, but I've over-done it here -- `#!/usr/bin/env bash` is indeed better here, and not just for the sake of Macs.

Everything else is good on Mac? (In the "we've only one user" spirit I'd not been thinking much about it -- but I should!)

Reply:
Thanks, and yes, everything works fine on Mac OS.